src/traces/parcoords/defaults.js

```diff
@@ -97,4 +100,8 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
     if(!Array.isArray(dimensions) || !dimensions.length) {
         traceOut.visible = false;
     }
+
+    coerceFont('labelfont', coerce, layout.font, attributes.labelfont.dflt || {});
```

Comment:
I could do that. The consequence is that the tick and dimension label fonts would be too large:
![image](https://cloud.githubusercontent.com/assets/1548516/25458491/6ce4f1b0-2ada-11e7-968e-692205e4f21f.png)

It's even worse if the user specified a larger font centrally, e.g. size 16:
![image](https://cloud.githubusercontent.com/assets/1548516/25458561/aae06ef4-2ada-11e7-8ef2-21ddb26435db.png)

Since I can go either way, pls confirm if I should preserve the current fontsize-moderation logic or use the simpler logic. Consistency matters a lot on one hand, and having ergonomically good defaults is also valuable.